llvm/lib/CodeGen/SelectionDAG/InstrEmitter.cpp

```diff
@@ -1191,6 +1191,21 @@ EmitMachineNode(SDNode *Node, bool IsClone, bool IsCloned,
     }
   }
 
+  if (Opc == TargetOpcode::CONVERGENCECTRL_GLUE) {
+    MIB->getOperand(0).setIsKill(false);
+  }
```

Comment:
Why do you need this? General you use clearKillFlags but I wouldn't expect that to be necessary in InstrEmitter

Reply:
I actually don't remember why I put it. Removed!